CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+This repository is a TC39 project and subscribes to its [code of conduct](https://tc39.es/code-of-conduct/). It is available at [https://tc39.es/code-of-conduct/](https://tc39.es/code-of-conduct/).
+
+To ask a question or report an issue, please email [tc39-conduct-reports@googlegroups.com](mailto:tc39-conduct-reports@googlegroups.com).
```

Comment:
not sure if we should provide this call to action here.  preferable for them to see the reporting guidelines on the CoC page.  thoughts @ljharb ?

Reply:
update:  I see that's what's in `CODE_OF_CONDUCT.md` in one or more repos, so I guess consistency is good?

Reply:
I'm fine with it either way, personally

Reply:
Ideally, I think, create a repository for the coc, and all repositories should have a minimal pointer to that repository (or more accurately, TC39 coc?).

Reply:
We have that repo already, and that's what this PR does, right?